### ISSUE TITLE
HDR wars - ads vs content?

### ISSUE THREAD

**Issue by greggman** [w3c/ColorWeb-CG#78]
Is there any concern that ads will use HDR to try to make their ads brighter than the content? The content then will be forced to increase all of their color values to brighter colors to try to make sure the ads don't stick out too much?

Is there a need to allow a page to disallow HDR in iframes? (maybe this already exists?)

I'm imaginig we'll start seeing something like

<img width="1435" alt="hdr-ads" src="https://user-images.githubusercontent.com/234804/173682337-e2bcf84b-58a6-4e63-b2e0-2375d1252270.png">

Or even just making the bright colors/text in the ads be brighter than anything outside of the ads

Note: I get ads can already flash, animate, play video, to be distracting so it's not entirely clear HDR is adding anything. On the other hand, maybe it is different if content has to be respond by multiplying all of their colors by 2x.

**Comment by karip** [comment]
I had a similar concern a while ago: https://github.com/w3c/csswg-drafts/issues/553#issuecomment-272411449 - maybe accessibility features will address this issue?

**Comment by smfr** [comment]
I disagree that accessibility options are the right approach to addressing this. HDR ads would be an annoyance to all/most users of the web, adding to the set of annoyances that each incrementally make the web less of a joy to use. Only some subset of users would take the effort to change their accessibility settings to tone down the HDR, and in doing so they would also limit their ability to benefit from content images where HDR is a good thing.

One option here is to prevent HDR in cross-origin iframes (perhaps with an `allow` property to enable it).
